heareval/tasks/util/luigi.py

```diff
@@ -17,10 +17,13 @@
 
 PROCESSMETADATACOLS = [
     "relpath",
+    # Like what is this and what is its contract?
```

Comment:
Can we please document ProcessMetadata and the contract behind its columns? There are several possible interpretations for what things are, how they are created, and what they mean? Could we just simplify this a lot?

Reply:
Ya I was thinking about this. I just it might sense to have different metadata templates for each dataset type: scene classification / multi-label, timestamp multi-label, etc...

Reply:
One option would be to create a base case for each type that have the csv template. SceneBasedMetadata and TimestampBasedMetadata or something. Define the columns for each in that. Could also serve as a place to document the requirements for ProcessMetadata tasks.